src/main/java/io/github/ImpactDevelopment/installer/target/targets/Forge.java

```diff
@@ -122,26 +127,6 @@ public String apply() {
             throw new RuntimeException(e);
         }
 
-        // Remove other Impact forge jars
```

Comment:
This is a REALLY big downside. People might not realize that it doesn't overwrite other jars, or might not realize that it'll crash if you have more than one. Maybe we can keep this, but just have it pop up a warning to them if there's another jar beginning with `Impact-` in the same folder?

Reply:
I can see the argument for a warning, but it's worth noting that they will have saved the file themselves in the mods/1.12.2 folder and so will have _seen_ the other mods when doing so.

Reply:
They won't know it's a problem, or won't realize that it doesn't overwrite